packages/suite/src/utils/wallet/coinmarket/coinmarketUtils.ts

```diff
@@ -491,3 +491,33 @@ export const coinmarketGetSectionActionLabel = (
 
     return 'TR_COINMARKET_SWAP';
 };
+
+interface GetAddressAndTokenFromAccountOptionsGroupProps {
+    address: string;
+    token: string | null;
+}
+
+export const getAddressAndTokenFromAccountOptionsGroupProps = (
+    selected: CoinmarketAccountOptionsGroupOptionProps | undefined,
+): GetAddressAndTokenFromAccountOptionsGroupProps => {
+    if (!selected) {
+        return { address: '', token: null };
+    }
+
+    const networkSymbol = cryptoIdToNetworkSymbol(selected.value);
+    const networkType = networkSymbol ? getNetworkType(networkSymbol) : null;
+
+    // set token address for ERC20 transaction to estimate the fees more precisely
```

Comment:
that is true for changely but not for 1inch

Reply:
Yes. The new flow is in progress.